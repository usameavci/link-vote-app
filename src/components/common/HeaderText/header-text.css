.common-header-text span:nth-child(1) {
  font-weight: 700;
}

.common-header-text span:nth-child(2) {
  font-weight: 300;
}

.common-header-text span:nth-child(3) {
  font-weight: 400;
}
