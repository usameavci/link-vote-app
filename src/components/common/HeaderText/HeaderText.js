import React from "react";

import "./header-text.css";

const HeaderText = () => {
  return (
    <div data-testid="common-header-text" className="common-header-text">
      <span>Link</span>
      <span>VOTE</span>
      <span>Challenge</span>
    </div>
  );
};

export default HeaderText;
