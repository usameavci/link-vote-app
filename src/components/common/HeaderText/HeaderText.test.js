import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";

import HeaderText from "./HeaderText";

describe("HeaderText component tests", () => {
  it("renders without crashing", async () => {
    const component = <HeaderText />;

    const { getByTestId } = render(component);

    expect(getByTestId("common-header-text")).toBeInTheDocument();
  });

  it("matches snapshots", () => {
    const component = <HeaderText />;
    const tree = renderer.create(component);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
