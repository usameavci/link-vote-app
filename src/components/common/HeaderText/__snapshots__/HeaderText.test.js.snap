// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderText component tests matches snapshots 1`] = `
<div
  className="common-header-text"
  data-testid="common-header-text"
>
  <span>
    Link
  </span>
  <span>
    VOTE
  </span>
  <span>
    Challenge
  </span>
</div>
`;
